features/checkout/paying_for_order/preventing_not_available_payment_method_selection.feature

```diff
@@ -0,0 +1,21 @@
+@checkout
+Feature: Preventing not available payment method selection
+    In order to pay for my order properly
+    As a Customer
+    I want to be prevented from selecting not available payment methods
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store allows paying with "Offline"
+        And the store ships everywhere for free
+        And I am logged in as customer
+
+    @ui
+    Scenario: Not being able to select disabled payment method
+        Given the store has disabled "Express" payment method
```

Comment:
Maybe we could simplify this, by creating the method in background, but here saying it has been disabled?

```
Background:
    Given the store allows paying with "PayPal Express"

Scenario:
    Given the store has disabled payment method "PayPal Express"
```

Step in scenario set enabled to false and does not create a whole new payment method. WDYT?